Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.cpp

```diff
@@ -85,35 +84,51 @@ ActsExamples::VertexPerformanceWriter::VertexPerformanceWriter(
     throw std::bad_alloc();
   } else {
     // I/O parameters
+    m_outputTree->Branch("truthX", &m_truthX);
+    m_outputTree->Branch("truthY", &m_truthY);
+    m_outputTree->Branch("truthZ", &m_truthZ);
+    m_outputTree->Branch("truthT", &m_truthT);
+
+    m_outputTree->Branch("recoX", &m_recoX);
+    m_outputTree->Branch("recoY", &m_recoY);
+    m_outputTree->Branch("recoZ", &m_recoZ);
+    m_outputTree->Branch("recoT", &m_recoT);
+
     m_outputTree->Branch("resX", &m_resX);
     m_outputTree->Branch("resY", &m_resY);
     m_outputTree->Branch("resZ", &m_resZ);
+    m_outputTree->Branch("resT", &m_resT);
 
     m_outputTree->Branch("pullX", &m_pullX);
     m_outputTree->Branch("pullY", &m_pullY);
     m_outputTree->Branch("pullZ", &m_pullZ);
-
-    m_outputTree->Branch("recoX", &m_recoX);
-    m_outputTree->Branch("recoY", &m_recoY);
-    m_outputTree->Branch("recoZ", &m_recoZ);
+    m_outputTree->Branch("pullT", &m_pullT);
 
     m_outputTree->Branch("truthX", &m_truthX);
     m_outputTree->Branch("truthY", &m_truthY);
     m_outputTree->Branch("truthZ", &m_truthZ);
+    m_outputTree->Branch("truthT", &m_truthT);
```

Comment:
Are these duplicated? I see them in line 87-90 as well.

Reply:
nice catch - thanks